R/pk-parameter.R

```diff
@@ -1,29 +1,70 @@
+#' Standard PK-Parameters types defined in OSPSuite
+#' This is only used to defined how a user defined PK Parameter should be calculated
+#' @include enum.R
+#' @export
+StandardPKParameter <- enum(c(

```

Comment:
@abdullahhamadeh This enum should not be used as I previously stated. This is just used to know "How" a parameter should be calculated e.g. (cCmax, is max of conc, Auc is Aread under the curve etc..)